src/main/java/net/atos/zac/signalering/event/SignaleringEvent.java

```diff
@@ -57,6 +57,6 @@ public String getActor() {
 
     @Override
     public String toString() {
-        return String.format("%s %s", super.toString(), getActor());
+        return String.format("%s, actor: %s", super.toString(), getActor());
```

Comment:
sure this does not break anything in the frontend? no idea but worth checking?

Reply:
Guess not since events are backend only things right?

Reply:
I'll revert this just in case